src/AdyenNativeModules.tsx

```diff
@@ -0,0 +1,170 @@
+import { NativeModule, NativeModules } from 'react-native';
+import { find, NATIVE_COMPONENTS } from './ComponentMap';
+import {
+  ErrorCode,
+  LINKING_ERROR,
+  UNKNOWN_PAYMENT_METHOD_ERROR,
+} from './Core/constants';
+import {
+  Card,
+  PaymentAction,
+  PaymentMethod,
+  PaymentMethodsResponse,
+} from './Core/types';
+
+/** Universal interface for Adyen Native payment component */
+interface AdyenComponent {
+  /** Show component above current screen. */
+  open: (paymentMethods: PaymentMethodsResponse, configuration: any) => void;
+
+  /** Dismiss component from screen. */
+  hide: (success: boolean, message: string | undefined) => void;
```

Comment:
As we can see in exemple

```ts
nativeComponent.hide(true, { message: 'Completed' });
```

shouldn't the type be something like this ?

```suggestion
  hide: (success: boolean, option?: { message?: string }) => void;
```

Reply:
With latest version on `develop` branch

`nativeComponent.hide(true, 'Completed');`  failed while `nativeComponent.hide(true,  {message: 'Completed' });` succeed

Reply:
Hey @gkueny 

Thanks for feedback!
This is good find.

Reply:
Fixed in [0a94c14](https://github.com/Adyen/adyen-react-native/pull/166/commits/0a94c14ce72c92e340cc2deb59a6b67c2751a636)